src/syscalls/deprecated_business_logic_syscall_handler.rs

```diff
@@ -498,7 +504,11 @@ where
     ) -> Result<Address, SyscallHandlerError> {
         match self.read_and_validate_syscall_request("get_contract_address", vm, syscall_ptr)? {
             DeprecatedSyscallRequest::GetContractAddress(_) => {}
-            _ => return Err(SyscallHandlerError::ExpectedGetContractAddressRequest),
+            _ => {
+                return Err(SyscallHandlerError::ExpectedStruct(
```

Comment:
I think it will be useful to also print the received struct here and in the rest of the places